stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -339,6 +366,7 @@ internal class PaymentSheetViewModel internal constructor(
                     application
                 ),
                 starterArgs,
+                Logger.real(),
```

Comment:
You definitely want to use the real logger?

Reply:
Good point. I think I'll use the noop logger until we have a better logging story. Maybe `enableLogging` should be part of `PaymentSheet.Configuration`?